### DISCUSSION TITLE
Can't view forge version

### DISCUSSION THREAD

**Discussion by Nathan12212** [Cyfrin/foundry-full-course-cu#1573] [Q&A] [answered]
Hey,

I want to view forge version, but it's error

`cromewar@Nathan:~$ forge --version
ZOE ERROR (from forge): zoeParseOptions: unknown option (--version)
ZOE library version 2013-02-16`

**Comment by EngrPips** [comment]
Hello @Nathan12212 , maybe do `foundryUp` then try run `forge version` again.

**Reply by Nathan12212** [reply]
thanks, but it's still not working

**Reply by EngrPips** [reply]
I run `forge --version` on my terminal it printed the version of foundry i have installed, so it is really hard for me to say what is wrong your end but a maybe a screenshot of when you run running and the result you get would give some more insight.

below is mine 

<img width="1025" alt="Screenshot 2024-04-05 at 1 37 20 AM" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/95760918/82949161-e53b-4987-a04a-88dacbcf6106">

Looking forward to hear from you.

**Reply by Nathan12212** [reply]
Here it's my terminal result:
![Screenshot 2024-04-05 154632](https://github.com/Cyfrin/foundry-full-course-f23/assets/143945987/4e8fb46e-0444-4164-8dd7-e4d264f006a1)

**Reply by EngrPips** [reply]
Hmmm you are using `wsl`, never used it my whole life and when people that use get errors it's always difficult for me to know whats going on. i would check with people that use it and get back to you with whatever i find.

**Reply by Nathan12212** [reply]
Okay, Thank you so much

**Reply by EngrPips** [reply, marked as answer]
So i asked someone that knows about `wsl` and they said you might be having conflicting `forge` entries in your bin. and here is also a similar issue as yours on https://ethereum.stackexchange.com/questions/135581/im-working-with-foundry-forge-seems-to-be-installed-but-i-get-zoe-error-whenev `ethereum stack exchange`. maybe you can read through the post and get a solution to your issue.

**Reply by Nathan12212** [reply]
thank you, now i can run foundry:
![Screenshot 2024-04-06 072309](https://github.com/Cyfrin/foundry-full-course-f23/assets/143945987/7c6328fa-120f-480e-bc8b-3ec7ce0e9345)

**Reply by EngrPips** [reply]
Awesome, i am glad you figured it out.

**Reply by bouchaib87** [reply]
i have same problem , can you help me please ?

**Reply by EngrPips** [reply]
> i have same problem , can you help me please ?

What is your problem sir?

**Reply by EngrPips** [reply]
> I think `WSL` is not that bad to use for windows user. Like i was also getting errors firstly, but as time passed i became comfortable with it by solving it and finding solution, now i am not getting any errors. However it is not good as `MAC OS` or `LINUX OS` preinstalled. But it can be helpful who use only windows.

Yeah things get easier with time.
